utils/spdx/src/main/kotlin/SpdxSimpleLicenseMapping.kt

```diff
@@ -30,23 +32,23 @@ object SpdxSimpleLicenseMapping {
     /**
      * The map of simple license names associated with their corresponding [SPDX license][SpdxLicense].
      */
-    internal val simpleLicenseMapping by lazy {
+    internal val simpleLicenseMapping: Map<String, SpdxLicense> by lazy {
         val resource = checkNotNull(javaClass.getResource("/simple-license-mapping.yml"))
-        yamlMapper.readValue<Map<String, SpdxLicense>>(resource)
+        yamlMapper.readValue(resource)
     }
 
     /**
      * The map of simple license names associated with their corresponding [SPDX expression][SpdxLicenseIdExpression].
      */
-    val simpleExpressionMapping by lazy {
+    val simpleExpressionMapping: SortedMap<String, SpdxLicenseIdExpression> by lazy {
```

Comment:
Could we instead use `Map<String, SpdxLicenseIdExpression>` for not exposing the mutability? (Or does the sub-type matter to the outside?)